app/labor/markdown_parser.rb

```diff
@@ -176,7 +178,7 @@ def escape_liquid_tags_in_codeblock(content)
   end
 
   def possibly_raw_tag_syntax?(array)
-    array.any? { |string| ["{", "}", "raw", "endraw", "----"].include?(string) }
+    array.any? { |string| RAW_TAG_DELIMITERS.include?(string) }
```

Comment:
@rhymes Instead of iterating and doing an include, we can use the fact that arrays have set operations defined on them:

```suggestion
    (RAW_TAG_DELIMITERS & array).any?
```

Reply:
Cool!